### ISSUE TITLE
Perform gap analysis on CP senders for migrating to the UP

### ISSUE THREAD

**Issue by chris-kuryak** [CDCgov/prime-reportstream#12756]
## Problem statement
We need to develop a timeline that efficiently adds CP senders to the UP.
This work was dependent on the gap analysis.

## What you need to know
Goal: Helping develop a prioritization list and timeline that can be achieved over the next year for adding CP receivers to the UP.

## Scope
- Existing CP senders, not any new senders
- Using existing Engagement resources/capacity

## Documentation
- [UP Sender Migration Research Findings](https://docs.google.com/document/d/1gyFszhQ7aV9i7-z3nBmy8sLYLXr_uU9Y-HfJSDFEDQQ/edit#heading=h.mtc4swvlio8d) includes what work (assets, content, etc) may be needed
- [CP to UP Migration Plan presentation](https://cdc.sharepoint.com/:p:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B580c2466-f03c-4c0c-b8a2-82a604e88928%7D&action=edit&wdPreviousSession=eb95b9c8-48f2-858e-3fb8-efb7062f16de)
- [Sender Audit](https://docs.google.com/spreadsheets/d/1GerMI0hKSRK5svbcNzScUBHb5zJImLOixTpBOZXOdvU/edit#gid=443493841)

## Acceptance criteria
- [ ] Perform gap analysis on 5 senders
- [ ] Develop a method for determining if a sender is a high, medium, or low lift for migration from CP to UP
- [ ] Reviewed with Product and Engineering for alignment

**Comment by chris-kuryak** [comment]
This work is currently blocked by Platform mapping work.

**Comment by chris-kuryak** [comment]
Closing. Duplicate of #13011
